docs/kubernetes/Policy.md

```diff
@@ -0,0 +1,133 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.13.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Calico Policy for Kubernetes
+The Calico CNI plugin for Kubernetes allows you to specify network policy in the Kubernetes API using annotations.  
+> *Note*: annotation-based policy is currently experimental and is subject to change in future releases. 
+
+## Prerequisites
+* A Kubernetes v1.1 Deployment using the Calico CNI plugin.
+* You must be using the iptables kube-proxy in your deployment. This is the default proxy mode in Kubernetes v1.1.3.
+
+## Behavior
+Without annotation-based policy enabled, Calico follows the [Kubernetes networking model][k8s-network-model], allowing full connectivity between pods.
+
+When Calico's annotation-based policy is enabled: 
+- Pods will be, by default, isolated by namespace boundaries. Only pods in the same namespace can communicate.
+- Annotations can be used to expose access to pods outside of their namespace. 
+- Annotations can be used to futher isolate pods within their namespace.
+- Pods in the `kube-system` namespace (such as SkyDNS), are accessible to the rest of the cluster.  
+
+Since pods are, by default, isolated by namespace boundaries, they will:
+- not be accessible by pods outside of their namespace unless explicitly allowed via annoations.  
+- not be accessible via Kubernetes service IPs, NodePort services, or LoadBalancer services unless specificaly allowed using an annotation.
+- not be accessible by the compute hosts in your cluster unless specificaly allowed using an annotation. 
+
+## Enabling annotation-based policy
+To enable annotation-based policy, add the `policy` section to your CNI network config file as shown.
+```
+$ cat /etc/cni/net.d/10-calico.conf
+{
+    "name": "calico-k8s-network",
+    "type": "calico",
+    "etcd_authority": "<ETCD_IP:ETCD_PORT>",
+    "log_level": "info",
+    "ipam": {
+        "type": "calico-ipam"
+    },
+    "policy": {
+        "type": "k8s-annotations",
+        "k8s_api_root": "<KUBERNETES_API_ROOT>",
+        "k8s_auth_token": "<AUTH_TOKEN>"
+    }
+}
+```
+
+The following configuration optons are supported in the `policy` section:
+- `type`: The type of policy to use.  Currently, only `k8s-annotations` is supported.
+- `k8s_api_root`: (Optional) Location of the Kubernetes API.  Default: `https://10.100.0.1:443/api/v1/`
```

Comment:
I don't think it would - that is the service VIP for the Kubernetes API, which I believe is authenticated even when TLS is disabled (could be wrong though, so I'll double check).